benchmarking/smartseq2/merge_picard_metrics/merge_picard_metrics.wdl

```diff
@@ -0,0 +1,64 @@
+task MergePicardMet {
```

Comment:
I think you should change `Met` or `Mets` to `Metric` or `Metrics` to be consistent across the PR